src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -58,14 +58,17 @@ enum Kind {
     private final String className;
     private final String superName;
     private final ClassSourceBuilder enclosing;
+    private final String runtimeHelperName;
 
-    ClassSourceBuilder(SourceFileBuilder builder, String modifiers, Kind kind, String className, String superName, ClassSourceBuilder enclosing) {
+    ClassSourceBuilder(SourceFileBuilder builder, String modifiers, Kind kind, String className, String superName,
+                       ClassSourceBuilder enclosing, String runtimeHelperName) {
```

Comment:
nit: maybe we want to call this "mainHeaderClass" or something like that?

Reply:
I had `firstHeaderName` at first, but I didn't like how that name doesn't signal what the purpose of the field is. So, I switched it to `runtimeHelperName` (there might be a better name). If we ever decide to change the location of the helper methods again, the name would still work, since the purpose stays the same.